File: VS Projects/Basic/Calculator/Calculator/Form1.vb
Public Class Form1
    Private Sub Form1_Load(sender As Object, e As EventArgs)

    End Sub

    Private Sub clr_Click(sender As Object, e As EventArgs)
        TextBox1.Clear()
        TextBox2.Clear()
        TextBox3.Clear()

    End Sub

    Private Sub Plus_Click_1(sender As Object, e As EventArgs)
        Dim num1, num2, calc As Integer
        num1 = Convert.ToInt32(TextBox1.Text)
        num2 = Convert.ToInt32(TextBox2.Text)
        calc = num1 + num2
        TextBox3.Text = calc.ToString()
    End Sub

    Private Sub Minus_Click_1(sender As Object, e As EventArgs)
        Dim num1, num2, calc As Integer
        num1 = Convert.ToInt32(TextBox1.Text)
        num2 = Convert.ToInt32(TextBox2.Text)
        calc = num1 - num2
        TextBox3.Text = calc.ToString()
    End Sub

    Private Sub Multiply_Click_1(sender As Object, e As EventArgs)
        Dim num1, num2, calc As Integer
        num1 = Convert.ToInt32(TextBox1.Text)
        num2 = Convert.ToInt32(TextBox2.Text)
        calc = num1 * num2
        TextBox3.Text = calc.ToString()
    End Sub

    Private Sub Divide_Click_1(sender As Object, e As EventArgs)
        Dim num1, num2, calc As Integer
        num1 = Convert.ToInt32(TextBox1.Text)
        num2 = Convert.ToInt32(TextBox2.Text)
        calc = num1 / num2
        TextBox3.Text = calc.ToString()
    End Sub

    Private Sub ModulusButton_Click_1(sender As Object, e As EventArgs)
        Dim num1, num2, calc As Integer
        num1 = Convert.ToInt32(TextBox1.Text)
        num2 = Convert.ToInt32(TextBox2.Text)
        calc = num1 Mod num2
        TextBox3.Text = calc.ToString()
    End Sub

End Class

File: VS Projects/C#/Count/Count/Form1.cs
namespace Count
{
    public partial class Form1 : Form
    {
        double sum = 0;
        public Form1()
        {
            InitializeComponent();
        }

        private void b1_Click(object sender, EventArgs e)
        {

            for (double i = 0; i <= 100; i++)
            {
                if (i == 100) {
                    sum += i;
                    tb.Text += $"{sum} "; 
            }else
                {
                    sum += i;
                    tb.Text += $"{sum}, ";
                }
                
            }
        }

        private void clr_Click(object sender, EventArgs e)
        {
            tb.Clear();
            sum = 0;
        }
    }
}